framework/include/indicators/ElementIndicator.h

```diff
@@ -52,7 +52,7 @@ class ElementIndicator : public Indicator,
   const MooseArray<Real> & _JxW;
   const MooseArray<Real> & _coord;
 
-  MooseVariable & _var;
+  // MooseVariable & _var;
```

Comment:
remove this?

Reply:
Or make it `MooseVariableField<Real>` - is probably the right solution (and don't forget to initialize it in the `.C` file.

Reply:
I did the second one, and then used it for `_u` etc.